inference-engine/src/mkldnn_plugin/nodes/mkldnn_split_node.cpp

```diff
@@ -36,26 +35,36 @@ void MKLDNNSplitNode::getSupportedDescriptors() {
 }
 
 void MKLDNNSplitNode::initSupportedPrimitiveDescriptors() {
```

Comment:
In general all initSupportedPrimitiveDescriptors method looks overcomplicated a little bit:
1) Initialization of plain, per channel and blocked descriptors is implemented differently. First two are created using predefined list of memory formats (but for plain format it is united with dims validation). Blocked is initialized via custom BlockingDesc. Both ways are ok, but the second one seems to be more generic. For me it makes sense to consider second way for plain and per channel layouts. 
2) Logic for initialization of optimized inplace case and optimized blocked inplace cases is almost the same. Can it be united and maybe extracted into separate method?

Reply:
The code has been refactored to reflect these suggestions in ee8234d71967059ae0d72418abd7796120119305.